spec/story/story.spec.js

```diff
@@ -0,0 +1,211 @@
+        client.setBaseUrl('http://api-test.asyncvoter.agileventures.org');
+        client.createStory(newStory, function (err, data, response) {
+            testResponse = response;
+            testData = data;
+            testErr = err;
+            done();
+        });
+    });
+
+    it('should return a code 200', function () {
+        expect(testResponse.statusCode).to.equal(200);
+    });
+
+    it('should return a null error', function () {
+        expect(testErr).to.equal(null);
+    });
+
+    it('should return a story with correct name', function () {
+        expect(testData.name).to.equal(newStory.name);
+    });
+
+    it('should return a story with correct source', function () {
+        expect(testData.source).to.equal(newStory.source);
+    });
+
+    it('should return a story with correct url', function () {
+        expect(testData.url).to.equal(newStory.url);
+    });
+
+    it('should return a story with correct userId', function () {
+        expect(testData.userId).to.equal(newStory.userId);
+    });
+});
+
+describe('testing updating a voter story', function () {
+    var storyId;
+    var newStory = {
+        name: 'Update API Client',
+        source: '#async_voter_test',
+        userId: '@test_user',
+        url: 'https://example.com/update_api_client'
+    };
+    before(function (done) {
+        this.timeout(2500);
+        client.setBaseUrl('http://api-test.asyncvoter.agileventures.org');
+        client.createStory(newStory, function (err, data, response) {
+            storyId = data._id;
+            done();
+        });
+    });
+
+    it('should update story with a size and return a code 200', function (done) {
+        var updates = {size: '1'};
+        client.updateStory(storyId, updates, function (err, data, response) {
+            expect(response.statusCode).to.equal(200);
+            expect(data.size).to.equal('1');
+            done();
+        });
+    });
+
+    it('should change the existing size of a story and return a code 200', function (done) {
+        var updates = {size: '3'};
+        client.updateStory(storyId, updates, function (err, data, response) {
+            expect(response.statusCode).to.equal(200);
+            expect(data.size).to.equal('3');
+            done();
+        });
+    });
+
+});
+
+describe('testing getting a voter story', function () {
+    var storyId;
+    var newStory = {
+        name: 'Get API Story',
+        source: '#async_voter_story',
+        userId: '@story_user',
+        size: '2',
+        url: 'https://example.com/get_api_story'
+    };
+    before(function (done) {
+        this.timeout(2500);
+        client.setBaseUrl('http://api-test.asyncvoter.agileventures.org');
+        client.createStory(newStory, function (err, data, response) {
+            storyId = data._id;
+            done();
+        });
+    });
+
+    it('should return a code 200', function (done) {
+        client.getStory(storyId, function (err, data, response) {
```

Comment:
in this test and below we could refactor to get all the data up front and then test with one liners, like we do at the top of the file